### PR TITLE
Add references to UnmanagedFunctionPointerAttribute

### PR DESCRIPTION
**Pull request by nxtn**
Contributes to dotnet/docs#11396

### CODE REVIEW

File: xml/System.Runtime.InteropServices/Marshal.xml [hunk 1]
```diff
@@ -3548,7 +3548,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The delegate `d` is converted to a function pointer that can be passed to unmanaged code using the [__stdcall](https://msdn.microsoft.com/library/e212594b-1827-4d07-9527-7d412b300df8) calling convention.  
+ The delegate `d` is converted to a function pointer that can be passed to unmanaged code using the [__stdcall](https://docs.microsoft.com/cpp/cpp/stdcall) calling convention. You can set the calling convention by applying the <xref:System.Runtime.InteropServices.UnmanagedFunctionPointerAttribute> to the delegate.
```
Comment (rpetrusha) [inline xml/System.Runtime.InteropServices/Marshal.xml]:
```suggestion
 The delegate `d` is converted to a function pointer that can be passed to unmanaged code using the [__stdcall](/cpp/cpp/stdcall) calling convention. You can set the calling convention by applying the <xref:System.Runtime.InteropServices.UnmanagedFunctionPointerAttribute> to the delegate.  
```

File: xml/System.Runtime.InteropServices/Marshal.xml [hunk 2]
```diff
@@ -3611,7 +3611,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The delegate `d` is converted to a function pointer that can be passed to unmanaged code by using the [__stdcall](https://msdn.microsoft.com/library/e212594b-1827-4d07-9527-7d412b300df8) calling convention.  
+ The delegate `d` is converted to a function pointer that can be passed to unmanaged code by using the [__stdcall](https://docs.microsoft.com/cpp/cpp/stdcall) calling convention. You can set the calling convention by applying the <xref:System.Runtime.InteropServices.UnmanagedFunctionPointerAttribute> to the delegate.
```
Comment (rpetrusha) [inline xml/System.Runtime.InteropServices/Marshal.xml]:
```suggestion
 The delegate `d` is converted to a function pointer that can be passed to unmanaged code by using the [__stdcall](/cpp/cpp/stdcall) calling convention. You can set the calling convention by applying the <xref:System.Runtime.InteropServices.UnmanagedFunctionPointerAttribute> to the delegate.  
```

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @NextTurn. I'll merge your PR now.

**Comment by NextTurn** [comment]
For more information, see https://github.com/dotnet/coreclr/blob/f1d2cd463508a3fe437043979673fec427c8b0ff/src/vm/dllimport.cpp#L2782-L2783

**Pull request merged**